app/src/main/res/layout/previous_responses_header_item.xml

```diff
@@ -59,7 +59,7 @@
       android:paddingEnd="8dp"
       android:text="@{viewModel.computePreviousResponsesHeaderText()}"
       android:textAllCaps="true"
-      android:textColor="@color/color_def_mid_grey"
+      android:textColor="@color/component_color_shared_previous_responses_header_item_container_background_color"
```

Comment:
This is not a `background_color` its a `text_color`. Rename it accordingly.

Reply:
Done